recipes/omniscidb-cpu/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "omniscidb-cpu" %}
+{% set version = "4.6.1" %}
+{% set sha256 = "04d69fd4b5d48e4bff937ad9c149c8a98746d54263485fcc3b840a4682bdda0d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/omnisci/omniscidb/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}   # required for C++ headers
+    - cmake 3.12.*
+    - make
+    - maven
+    - openjdk 8.*
+    - wget
+    - clangdev 7.*
+    - go >=1.6
+    - bisonpp
+  host:
+    - snappy  # required for mapd-core > 4.5.0
+    - arrow-cpp 0.12.*
+    - blosc
+    - boost-cpp >=1.68
```

Comment:
This should be taken care of by the global boost pinning

Reply:
We did that because we had some issues with the newest boost-cpp .. I am trying without this pinning to check if it is working now ..